pnpm - Missing peer on install

<!-- ⚠️ Please DON'T ignore the issue template -->

<!-- 💡 Consider upgrading to the latest version before sending the issue -->

**Describe the bug**
pnpm fails installation

**To Reproduce**
Steps to reproduce the behavior:
 `pnpm create slidev`

Error message:
```
ERR_PNPM_PEER_DEP_ISSUES  Unmet peer dependencies

.
└─┬ @slidev/cli
  └─┬ @slidev/client
    └─┬ @vueuse/motion
      └── ✕ missing peer @nuxt/kit@npm:@nuxt/kit-edge@latest
Peer dependencies that should be installed:
  @nuxt/kit@npm:@nuxt/kit-edge@latest  
```

**Desktop (please complete the following information):**
 - OS: Fedora 36
 - Slidev version: 0.33.0

--- a/packages/create-app/template/_npmrc
+++ b/packages/create-app/template/_npmrc
@@ -1,2 +1,3 @@
 # for pnpm
 shamefully-hoist=true
+auto-install-peers=true

